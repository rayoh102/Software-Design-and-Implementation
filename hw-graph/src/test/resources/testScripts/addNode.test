# Tests node addition

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# See if node is in graph
ListNodes graph1