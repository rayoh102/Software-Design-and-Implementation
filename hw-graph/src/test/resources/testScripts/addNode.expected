# Tests node addition

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# See if node is in graph
graph1 contains: n1